frontend/src/antd.less

```diff
@@ -16,7 +16,7 @@ with SASS, leading to duplicated config. When changing any variable here, please
 @disabled-color: @text-muted;
 @border-radius-base: 2px;
 @border-color-base: #d9d9d9;
-@body-background:  #f2f2f2;
+@body-background:  #fff;
```

Comment:
Context on this change?

Reply:
Otherwise you get a flash of light gray when loading the page